app/products/calls/types/calls.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Perhaps we can make this file `calls.d.ts`

Reply:
Ah, I reverted that change. `calls.d.ts` is causing some troubles with importing. I'm going to leave that for later since it's not crucial.